build/Common.test.props

```diff
@@ -23,16 +23,12 @@
     <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
   </ItemGroup>
 
-  <PropertyGroup Condition="$(MSBuildProjectName.EndsWith('.Tests'))">
+  <PropertyGroup Condition="$(MSBuildProjectName.Contains('.Tests'))">
```

Comment:
There are test projects ending in `Win` perhaps I will change this to be any project under `test/**`.